styles/globals.css

```diff
@@ -2,3 +2,48 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/***** start of styles for the home page main widget svg *****/
+.st0 {
+  display: none;
+}
+.st1 {
+  display: inline;
+}
+.st2 {
+  clip-path: url(#SVGID_2_);
+}
+.st3 {
+  fill: #ff867f;
+}
+.st4 {
+  fill-rule: evenodd;
+  clip-rule: evenodd;
+  fill: #c50e29;
+}
+.st5 {
+  fill-rule: evenodd;
+  clip-rule: evenodd;
+  fill: #fefefe;
+}
+.st6 {
+  fill-rule: evenodd;
+  clip-rule: evenodd;
+  fill: #fffdfc;
+}
+.st7 {
+  fill: #ffffff;
+}
+.st8 {
+  font-family: "MyriadPro-Regular";
+  fill: white;
+}
+.st9 {
+  font-size: 3px;
+}
+.st10 {
+  display: none;
+  fill: #ffffff;
+}
```

Comment:
Since we have tailwind and Chakra, is there a chance we can remove those and use them?